fontforge/tottf.c

```diff
@@ -3809,23 +3814,24 @@ void DefaultTTFEnglishNames(struct ttflangname *dummy, SplineFont *sf) {
     if ( dummy->names[ttf_subfamily]==NULL || *dummy->names[ttf_subfamily]=='\0' )
 	dummy->names[ttf_subfamily] = utf8_verify_copy(SFGetModifiers(sf));
     if ( dummy->names[ttf_uniqueid]==NULL || *dummy->names[ttf_uniqueid]=='\0' ) {
-	time(&now);
-	tm = localtime(&now);
-	sprintf( buffer, "%s : %s : %d-%d-%d",
-		BDFFoundry?BDFFoundry:TTFFoundry?TTFFoundry:"FontForge 2.0",
-		sf->fullname!=NULL?sf->fullname:sf->fontname,
-		tm->tm_mday, tm->tm_mon+1, tm->tm_year+1900 );
+    int major = -1;
+    int minor = -1;
+	sprintf( buffer, "%s;%s;%s %s",
+        versionStr,
+		BDFFoundry?BDFFoundry:TTFFoundry?TTFFoundry:"UKWN",
```

Comment:
I think that the original code works around the fact that FontForge does not come with its default TTFFoundry set to FontForge. If we want to drop the work-around, we need to fix the problem. I'm no TrueType expert; is "FontForge 2.0" a correct default value?